clients/py/v3io_frames/pbutils.py

```diff
@@ -141,18 +157,25 @@ def df2msg(df, labels=None, index_cols=None):
         else:
             serieses = [idx2series(df.index)]
 
-        indices = [series2col(s) for s in serieses]
+        indices = [series2col(s, s.name) for s in serieses]
+
+    columns = []
+    for name in df.columns:
+        series = df[name]
+        if isinstance(name, int):
```

Comment:
isn't for every column named as an int you will prefix it with `column_`?
and then when you read the DF you will get columns: `column_23, column_5` instead of `23,5`.
is that what we want?

Reply:
Added a check.